code_setup.sh

```
@@ -440,6 +440,61 @@ done
+		deployFile ${SRC_FILE_PATH} ${DEST_FILE_PATH} true; #over-write
+		checkResult $?;
+	elif [[ -d "$SRC_FILE_PATH" ]]; then
+		deployDirectory ${SRC_FILE_PATH} ${DEST_FILE_PATH} true; #DO over-write
+		checkResult $?;
+	else #WTF
+		echo "> File to deploy '$FILENAME' ($SRC_FILE_PATH) is neither a directory or a file!";
+		ls -l $FILENAME;
+		exit 1;
+	fi
+done
+mapfile -t SUBMODULES < <(git submodule foreach --quiet 'basename "$(pwd)"');
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)